packages/css/src/components/logo/README.md

```diff
@@ -34,7 +34,7 @@ The sub-brands are:
 - The logo links to the homepage of the website or application.
 - If the application is a form, application, or tool without a homepage, the logo links to the page where the form, application, or tool is referred to.
 
-The height of the logo is always 40 pixels.
+The logo is 40 pixels tall at its minimum, growing to 56 pixels in wider windows.
```

Comment:
This also requires some (complicated) changes in Figma, I'll reopen the logo variables ticket.